util/network/Cargo.toml

```diff
@@ -34,6 +34,7 @@ ethcore-logger = { path ="../../logger" }
 ipnetwork = "0.12.6"
 hash = { path = "../hash" }
 serde_json = "1.0"
+error-chain = { version = "0.11", default-features = false }
```

Comment:
Why we don't want default features? Is it because we print backtrace in other places?

Reply:
Yes, we don't want backtraces (at least for now). It may change in future and we may add them to the debug builds